psse/psse-model/src/main/java/com/powsybl/psse/model/PsseTransformer.java

```diff
@@ -0,0 +1,618 @@
+            return ckt;
+        }
+
+        public void setCkt(String ckt) {
+            this.ckt = ckt;
+        }
+
+        public int getCw() {
+            return cw;
+        }
+
+        public void setCw(int cw) {
+            this.cw = cw;
+        }
+
+        public int getCz() {
+            return cz;
+        }
+
+        public void setCz(int cz) {
+            this.cz = cz;
+        }
+
+        public int getCm() {
+            return cm;
+        }
+
+        public void setCm(int cm) {
+            this.cm = cm;
+        }
+
+        public double getMag1() {
+            return mag1;
+        }
+
+        public void setMag1(double mag1) {
+            this.mag1 = mag1;
+        }
+
+        public double getMag2() {
+            return mag2;
+        }
+
+        public void setMag2(double mag2) {
+            this.mag2 = mag2;
+        }
+
+        public int getNmetr() {
+            return nmetr;
+        }
+
+        public void setNmetr(int nmetr) {
+            this.nmetr = nmetr;
+        }
+
+        public String getName() {
+            return name;
+        }
+
+        public void setName(String name) {
+            this.name = name;
+        }
+
+        public int getStat() {
+            return stat;
+        }
+
+        public void setStat(int stat) {
+            this.stat = stat;
+        }
+
+        public int getO1() {
+            return o1;
+        }
+
+        public void setO1(int o1) {
+            this.o1 = o1;
+        }
+
+        public double getF1() {
+            return f1;
+        }
+
+        public void setF1(double f1) {
+            this.f1 = f1;
+        }
+
+        public int getO2() {
+            return o2;
+        }
+
+        public void setO2(int o2) {
+            this.o2 = o2;
+        }
+
+        public double getF2() {
+            return f2;
+        }
+
+        public void setF2(double f2) {
+            this.f2 = f2;
+        }
+
+        public int getO3() {
+            return o3;
+        }
+
+        public void setO3(int o3) {
+            this.o3 = o3;
+        }
+
+        public double getF3() {
+            return f3;
+        }
+
+        public void setF3(double f3) {
+            this.f3 = f3;
+        }
+
+        public int getO4() {
+            return o4;
+        }
+
+        public void setO4(int o4) {
+            this.o4 = o4;
+        }
+
+        public double getF4() {
+            return f4;
+        }
+
+        public void setF4(double f4) {
+            this.f4 = f4;
+        }
+
+        public String getVecgrp() {
+            return vecgrp;
+        }
+
+        public void setVecgrp(String vecgrp) {
+            this.vecgrp = vecgrp;
+        }
+    }
+
+    public static class SecondRecord {
+
+        @Parsed(index = 0)
+        private double r12 = 0;
+
+        @Parsed(index = 1)
+        @Validate
+        private double x12;
+
+        @Parsed(index = 2)
+        private double sbase12 = Double.NaN;
```

Comment:
The default value is the whole system base in MVA, how to set it correctly ?

Reply:
After the second record ?

Reply:
Either at the conversion level or in a data model loading post processing.